src/actors/symbolication.rs

```diff
@@ -881,22 +882,41 @@ pub struct SymbolicateStacktraces {
     pub modules: Vec<CompleteObjectInfo>,
 }
 
+async fn measure_task_timeout<T, F>(task_name: &str, future: F, duration: Duration) -> F::Output
```

Comment:
_nit_: Can be moved into utils if we figure out how to capture timeouts generically. Fine for now.

Reply:
Agreed, yes. I wanted to avoid double-wrapping this for now, but will do that as a followup